spec/controllers/two_factor_authentication/piv_cac_verification_controller_spec.rb

```diff
@@ -9,11 +9,19 @@
 
   let(:nonce) { 'once' }
 
+  let(:x509_subject) { 'o=US, ou=DoD, cn=John.Doe.1234' }
+
   before(:each) do
-    allow(subject).to receive(:user_session).and_return(piv_cac_nonce: nonce)
+    session_info = {piv_cac_nonce: nonce}
```

Comment:
Looks like Code Climate didn't catch this missing space around the brackets.

Reply:
good catch. Rubocop didn't catch it either when I ran `make lint`.